pheps/phep-0001.md

```diff
@@ -0,0 +1,338 @@
+```
+PHEP: 1
+Title: PHEP Purpose and Guidelines
+Author: Jonathan Niehof <jtniehof@gmail.com> <https://orcid.org/0000-0001-6286-5809>
+Discussions-To: https://github.com/heliophysicsPy/standards/pull/22
+Status: Draft
+Type: Process
+Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+Created: 20-Oct-2023
+Post-History: 24-Oct-2023, 31-Oct-2023, 9-Nov-2023
+```
+
+# What is a PHEP?
+<a name="what-is-a-phep"></a>
+PHEP stands for PyHC Enhancement Proposal. A PHEP provides information to the Python in Heliophysics Community (PyHC), describes its processes, or establishes standards for its packages.
+
+PHEPs are the primary mechanism for proposing standards, collecting community input, and documenting the design decisions underlying PyHC standards and recommendations. The PHEP process develops and documents consensus within the community. The PHEP author is responsible for building this consensus and documenting dissenting opinions.
+
+Because PHEPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal. This record is available by the normal git commands for retrieving older revisions and can be browsed on GitHub.
```

Comment:
If you mean "turn 'browsed on GitHub' into a link to the standards repository", definitly...that was an oversight on my part.

For history of the draft, more involved linking might be tricky. The Discussion-To header would remain and link the PR so history can be browsed there--this does make me wonder if it's worth replacing Discussion-To with Pull-Request or something like that.

This does argue against doing a squash commit when a PHEP is Final, since we want to keep history. Maybe for discussion in #24 or maybe it belongs here, down where "editors update the preamble and merge the PR into `main`".

If you mean older revisions in the sense of the PHEP being revised after Final, if [immutability](https://github.com/heliophysicsPy/standards/pull/22#pullrequestreview-1695467345) is accepted that's handled by the `Replaces` and `Replaced-By` headers.

Reply:
Updated to [link repo](https://github.com/jtniehof/standards/blob/e335aacd1346f0876f5a54d9d617bf3c11ec2e32/pheps/phep-0001.md?plain=1#L19) and now there's an explicit statement to [retain history](https://github.com/jtniehof/standards/blob/e335aacd1346f0876f5a54d9d617bf3c11ec2e32/pheps/phep-0001.md?plain=1#L147) (i.e. no squash commit).